sympy/matrices/expressions/tests/test_diagonal.py

```diff
@@ -9,9 +9,18 @@
 d = DiagonalOf(X)
 
 def test_DiagonalMatrix():
+    x = MatrixSymbol('x', n, 1)
     assert D.shape == (n, n)
     assert D[1, 2] == 0
     assert D[1, 1] == x[1, 0]
```

Comment:
OH! I didn't notice that one because it was already there. Yes, that is definitely wrong. It should be `x[1,1]`.